src/components/Replies/ViewReplies/style.js

```diff
@@ -0,0 +1,22 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  reply: {
+    width: 'calc(100% - 48px)',
+    marginLeft: theme.spacing(6),
+    background: '#f6f7fa',
+  },
```

Comment:
add `border: '1px solid',  borderRadius: 5,` to the reply helps to add the border styling